package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@angular-material-components/datetime-picker": "^8.0.0",
     "@angular/animations": "~14.2.7",
-    "@angular/cdk": "^14.2.5",
+    "@angular/cdk": "^14.2.7",
     "@angular/common": "~14.2.7",
```

Comment:
Remove this change for now, since its not affecting the actual issue.

Reply:
Okay, I will work on these changes @Angamanga